src/XrdSciTokens/XrdSciTokensAccess.cc

```diff
@@ -29,6 +32,38 @@ XrdVERSIONINFO(XrdAccAuthorizeObjAdd, XrdAccSciTokens);
 
 namespace {
 
+enum LogMask {
```

Comment:
This has gotten a life of its own and I see the same implementation in three  other modules (XrdMacaroonsHandler.hh, XrdTpcTPC.hh, and XrdVomsMapfile.hh) making this the fourth sych implementation. Shouldn't this be abstracted into a utility function?